injestion/set_api_key.py

```
@@ -0,0 +1,38 @@
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+REDIS_HOST = os.getenv("REDIS_HOST")
+REDIS_PORT = os.getenv("REDIS_PORT")
+API_KEY = os.getenv("API_KEY")
+
+try:
+    REDIS_PORT = int(os.getenv("REDIS_PORT"))
+except ValueError:
+    raise ValueError("REDIS_PORT must be a valid integer")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)